test/test_jit.py

```diff
@@ -6716,6 +6748,114 @@ def func(x):
     # dtype=torch.long), torch.full((S,), S, dtype=torch.long), torch.randint(1,S,(S,), dtype=torch.long))),
 ]
 
+nn_module_tests = [
+    dict(
+        module_name='Threshold',
+        constructor_args=(2, 1),
+        input_size=(2, 3, 4, 5),
+    ),
+    dict(
+        module_name='ReLU',
+        input_size=(2, 3, 4, 5),
+    ),
+    dict(
+        module_name='ReLU6',
+        input_size=(2, 3, 4, 5),
+        check_inplace=True,
+    ),
+    dict(
+        module_name='RReLU',
+        input_size=(1, 2, 2),
+    ),
+    dict(
+        module_name='Hardtanh',
+        input_size=(3, 2, 5),
+    ),
+    dict(
+        module_name='Sigmoid',
+        input_size=(2, 3, 4, 5)
+    ),
+    dict(
+        module_name='Tanh',
+        input_size=(2, 3, 4, 5)
+    ),
+    dict(
+        module_name='ELU',
+        constructor_args=(2.,),
+        input_size=(3, 2, 5),
+    ),
+    dict(
+        module_name='SELU',
+        input_size=(3, 2, 5),
+    ),
+    dict(
+        module_name='GLU',
+        constructor_args=(1,),
+        input_size=(5, 6, 7),
+        desc='dim'
+    ),
+    dict(
+        module_name='Hardshrink',
+        constructor_args=(2.,),
+        input_size=(4, 3, 2, 4),
+    ),
+    dict(
+        module_name='LeakyReLU',
+        constructor_args=(0.5,),
+        input_size=(3, 2, 5),
+        desc='with_negval'
+    ),
+    dict(
+        module_name='LogSigmoid',
+        input_size=(2, 3, 4),
+    ),
+    dict(
+        module_name='Softplus',
+        constructor_args=(2, -100),
+        input_size=(10, 20),
+        desc='beta_threshold',
+    ),
+    dict(
+        module_name='Softshrink',
+        constructor_args=(1,),
+        input_size=(3, 2, 5),
+        desc='lambda',
+    ),
+    dict(
+        module_name='PReLU',
+        constructor_args=(3,),
+        input_size=(2, 3, 4),
+        desc='1d_multiparam',
+    ),
+    dict(
+        module_name='Softsign',
+        input_size=(3, 2, 5),
+    ),
+    dict(
+        module_name='Tanhshrink',
+        input_size=(2, 3, 4, 5)
+    ),
+    dict(
+        module_name='Softmin',
+        constructor_args=(1,),
+        input_size=(2, 3, 5, 10),
+    ),
+    dict(
+        module_name='Softmax',
+        constructor_args=(1,),
+        input_size=(10, 20),
+    ),
+    dict(
+        module_name='Softmax2d',
+        input_size=(1, 3, 10, 20),
+    ),
+    dict(
+        module_name='LogSoftmax',
+        constructor_args=(1,),
+        input_size=(10, 20),
+    ),
+]
```

Comment:
Those should be taken from `common_nn.py`.

Reply:
The reason I added it here is that:

- when running test_nn.py, it pops up several kvs (e.g. module_name) from the dict object in common_nn.py, make it cannot be used in test_jit.py. I don't want to break the behavior in test_nn so I added them here, but I will try remove the dict.pop from test_nn

- it contains test cases for other modules and multiple tests for each class. I can add the classes from other modules to blacklist.